app/assets/javascripts/discourse/components/pwa-install-banner.js.es6

```diff
@@ -0,0 +1,61 @@
+import {
+  default as computed,
+  on
+} from "ember-addons/ember-computed-decorators";
+
+const USER_DISMISSED_PROMPT_KEY = "dismissed-pwa-install-banner";
+
+export default Ember.Component.extend({
+  deferredInstallPromptEvent: null,
+
+  _handleInstallPromptEvent(event) {
+    // Prevent Chrome 76+ from automatically showing the prompt
+    event.preventDefault();
+    // Stash the event so it can be triggered later
+    this.set("deferredInstallPromptEvent", event);
+  },
+
+  @on("init")
```

Comment:
In general you'll want to attach events on `didInsertElement`, because it's after the component is inserted into the DOM.

You should also tear the event down in a `willDestroyElement` otherwise this event will leak in test mode where we tear up/down repeatedly.